gabarit/template_num/num_tests/functional_tests.py

```diff
@@ -58,14 +58,14 @@ def test01_CreateSamples(self):
         print("Test of the file utils/0_create_samples.py")
 
         # "Basic" case
-        basic_run = f"{activate_venv}python {full_path_lib}/test_template_num-scripts/utils/0_create_samples.py -f mono_class_mono_label.csv -n 15"
+        basic_run = f"{activate_venv}python {full_path_lib}/test_template_num-scripts/utils/0_create_samples.py --overwrite -f mono_class_mono_label.csv -n 15"
         self.assertEqual(subprocess.run(basic_run, shell=True).returncode, 0)
         self.assertTrue(os.path.exists(os.path.join(full_path_lib, 'test_template_num-data', 'mono_class_mono_label_15_samples.csv')))
         df = pd.read_csv(f"{full_path_lib}/test_template_num-data/mono_class_mono_label_15_samples.csv", sep=';', encoding='utf-8')
         self.assertEqual(df.shape[0], 15)
 
         # Double files
-        double_files_run = f"{activate_venv}python {full_path_lib}/test_template_num-scripts/utils/0_create_samples.py -f mono_class_mono_label.csv multi_class_mono_label.csv -n 2000"
+        double_files_run = f"{activate_venv}python {full_path_lib}/test_template_num-scripts/utils/0_create_samples.py --overwrite -f mono_class_mono_label.csv multi_class_mono_label.csv -n 2000"
```

Comment:
Why does the NUM template have updated functional tests but not the other templates ?

Reply:
Good point. In fact most of these  `--overwrite` are not mandatory. I can remove those that we do not need.